engine/language-server/src/main/java/org/enso/languageserver/package-info.java

```diff
@@ -0,0 +1 @@
+package org.enso.languageserver;
```

Comment:
Adding this file into otherwise fully Scala project (together with `frgaalJavaCompilerSetting`) generates `.enso-source*` files in project root and allows `enso4igv` module to recognize the project. Then the project can be opened in IGV, NetBeans or VSCode, debugged, edited.